pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -151,14 +157,15 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 	// do not trust data in the cache, compare with the current state
 	healthyLiveVotingMembers, err := c.getHealthyVotingMembers(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("could not list etcd healthy members: %w", err)
 	}
 
 	// scaling down invariant
-	if len(healthyLiveVotingMembers) < desiredControlPlaneReplicasCount {
-		klog.V(2).Infof("Ignoring scale down since the number of healthy live etcd voting members (%d) < desired number of control-plane replicas (%d) ", len(healthyLiveVotingMembers), desiredControlPlaneReplicasCount)
+	minTolerableQuorum := etcdcli.MinimumTolerableQuorum(desiredControlPlaneReplicasCount)
+	if len(healthyLiveVotingMembers) < minTolerableQuorum {
+		klog.V(2).Infof("ignoring scale down since the number of healthy live etcd members (%d) < minimum required to maintain quorum (%d) ", len(healthyLiveVotingMembers), minTolerableQuorum)
 		if time.Now().After(c.lastTimeScaleDownEventWasSent.Add(5 * time.Minute)) {
-			recorder.Eventf("ScaleDown", "Ignoring scale down since the number of healthy live etcd voting members (%d) < desired number of control-plane replicas (%d) ", len(healthyLiveVotingMembers), desiredControlPlaneReplicasCount)
+			recorder.Eventf("ScaleDown", "Ignoring scale down since the number of healthy live etcd members (%d) < < minimum required to maintain quorum (%d) ", len(healthyLiveVotingMembers), minTolerableQuorum)
```

Comment:
```suggestion
			recorder.Eventf("ScaleDown", "Ignoring scale down since the number of healthy live etcd members (%d) < minimum required to maintain quorum (%d) ", len(healthyLiveVotingMembers), minTolerableQuorum)
```

Reply:
good catch 👍🏽  thx :)